<launch>  
  <node name="xbow4x_node" pkg="xbow400" exec="xbow4x_node" output="screen">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baudrate" value="38400"/>
    <param name="broadcast_tf" value="true"/>
    <param name="frame_id" value="cbimu_frame"/>
    <param name="measurement_mode" value="a"/> <!-- Angle Mode: a Scaled Mode: c Voltage Mode: r -->
    <param name="message_mode" value="C"/> <!-- Continuous Mode: C Poll Mode: P -->
    <param from="$(find-pkg-share xbow400)/params/imu_params.yaml" />
  </node>
 </launch>
